backups/lastSave.json

```diff
@@ -0,0 +1 @@
+"2022-08-10T17:44:18.011Z"
```

Comment:
There are better places than the github repo to store the backups of the database. I suggest the CDN since we're already storing the images there. That would also make it easy to have a /backup slash command that creates a new backup at any time.

Reply:
I deleted it locally already! :)
& am going to work on the `/backup` command.